tests/pipeline/test_engine.py

```diff
@@ -1315,3 +1318,58 @@ def test_string_classifiers_produce_categoricals(self):
             columns=self.asset_finder.retrieve_all(self.asset_finder.sids),
         )
         assert_frame_equal(result.c.unstack(), expected_final_result)
+
+
+class PopulateInitialWorkspaceTestCase(WithConstantInputs, ZiplineTestCase):
```

Comment:
It looks like we're only testing terms with `window_length`s of 0 right now.  Can we add a test for aliasing a windowed term?  An example of such a term would be an instance of `SimpleMovingAverage`.